assets/scss/utilities/_hide.scss

```diff
@@ -28,9 +28,13 @@
 }
 
 .u-hidden\@medium {
-    display: block;
-
     @include bp(medium) {
         display: none;
     }
 }
+
+.u-hidden\@mobile {
```

Comment:
It would be good to have consistent logic here. Now `u-hidden\@medium` and `u-hidden\@mobile` works different, as one of them hide elements on given breakpoint and up, and the other below. This will cause a headache in the future.